content/_changelogs/12.0.0.md

```diff
@@ -11,6 +11,11 @@ _Released MM/DD/YYYY_
 - Cypress now throws an error if commands are invoked from inside a `.should()`
   callback. This previously resulted in unusual and undefined behavior; it is
   now explicitly an error.
+- The `.within()` command now throws an error if given more than one DOM element
```

Comment:
@BlueWinds Can you add a migration guide entry on how to resolve this error?

Reply:
Added migration guide.